### PR TITLE
January 2024 release notes

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_help/release_notes/2024.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+page_order: 0
+nav_title: 2024
+layout: dev_guide
+noindex: true
+guide_top_header: "2024 Release Notes"
+
+page_type: landing
+description: "This section is home to release notes for 2024."
+
+guide_menu_title: ""
```
Comment (bre-fitzgerald) [inline _docs/_help/release_notes/2024.md:11]: Missing a title?

**Comment by rachel-feinberg** [comment]
Should we add the notes to the top of this list [here](https://braze-docs-hq1pbx1qb-braze.vercel.app/docs/help/release_notes?redirected=true#most-recent/)?

File: _docs/_help/release_notes/2024.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_help/release_notes/2024.md:11]: Also the headings on this page are out of order, which has broken the TOC.

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]

**Review by bre-fitzgerald — APPROVED** [review]
🚀

**Pull request merged**